### DISCUSSION TITLE
CLI - "plots add" argument to search all subdirectories

### DISCUSSION THREAD

**Discussion by ruant** [Chia-Network/chia-blockchain#6104] [Ideas]
Would be very nice to allow all sub directories to be searched for plots too. Instead of having to add each single one.

Use case:
I mount all drives in `/mnt/plots/`
Instead of me having to do `chia plots add -d /mnt/plots/drive##/` for each drive, I could just add `/mnt/plots/`

**Comment by funnycubeguy** [comment]
is this still not possible?

**Reply by loppefaaret** [reply]
no, the harvester will not go into subdirectories

**Reply by firmicus** [reply]
and that's right

**Reply by room101-dev** [reply]
The proper way to do this is to list all your directory's in the config.yaml file, there is not limit to path's.

**Reply by ruant** [reply]
@room101-dev 
> The proper way to do this is to list all your directory's in the config.yaml file, there is not limit to path's.

It's just a hassle when you have a lot of directories, and new ones get added all the time.

**Reply by x-vibe** [reply]
just wanted to check whether this is still the case, please? We are now in October. This seems like a very simple ask and for people with lots of directories, this could be a huge efficiency gain...

**Reply by stefan-lange** [reply]
I do a `find -name *.plot` and add all folders containing plots via `chia plots add` on startup in my harvester docker container. It is not perfect, but works pretty well :)

**Comment by BrandtH22** [comment]
@ruant , the newest version(s) of chia has fixes in place to resolve this issue. Please download it from here: https://www.chia.net/downloads/

Since this issue has been resolved with the latest version(s) of chia we will be closing this ticket but if we have closed this ticket in error do not hesitate to reach out to us again with any followup questions, comments, or if the issue persists after an update.

```
Please note that upgrading major versions of chia generally requires the config.yaml to be recreated using these steps:
1. Stop chia
2. Rename the config or move it to a different directory
3. In CLI run chia init
4. Copy any custom settings from the old config to the new (plot drive directories, pool info, trusted node info, etc) - be careful here as yaml is very picky about formatting including those leading spaces
5. Start Chia
```

The best place to reach our support team is on Discord (https://discord.gg/chia) or by reopening this ticket.

**Reply by andymarden** [reply]
And the best way of finding out how to do this in the new release: telepathy?
